rosdep/python.yaml

```diff
@@ -5281,16 +5281,16 @@ python-xlrd:
   debian: [python-xlrd]
   fedora: [python-xlrd]
   ubuntu: [python-xlrd]
-python-xmlplain-pip:
+python-xmlschema:
```

Comment:
You're still removing the python-xmlplain-pip. Please maintain that key and add the new keys.